docs/unicode-generated-tests.asciidoc

```diff
@@ -68,10 +68,10 @@ MAKE SURE TO RUN `perl6 CollationTestGen.p6 --folder=MoarVM/UNIDATA/UCA/Collatio
 Then paste those failures (if they're ok into CollationTestGen.p6 in the %todo variable.
 
 .Version
-* Unicode v12.1
```

Comment:
If I remember `CollationTestGen.p6` correctly, there was one set of options to run tests for *all* collation tests, and a separate one (maybe it was without arguments?) that actually generated the test. Can you confirm you did both of them, and how many failures happened when you ran `perl6 CollationTestGen.p6 --folder=MoarVM/UNIDATA/UCA/CollationTest --test-only --should-test `?

Looking at the signature of `MAIN` in that file:
```
sub MAIN (Bool :$should-test = False, Bool :$should-generate = True, Bool :$test-only = False, Str:D :$folder = $default-folder) {
```

Reply:
It's been a month since I did it so the details are hazy. I'm fairly certain I did both. I tried to follow the directions and hints you left as well as I could. 

Just tried running that command in my branch. There were 72 failure in 202763 tests. Most (all?) of them didn't seem to involve any new characters so I figured they were old failures.

Filtering out the passing tests - only the failures:

# Failed test ' (𝅘̴𝅥) MUSICAL SYMBOL NOTEHEAD BLACK, COMBINING TILDE OVERLAY, MUSICAL SYMBOL COMBINING STEM unicmp  (𝅘̴𝅥𝅮) MUSICAL SYMBOL NOTEHEAD BLACK, COMBINING TILDE OVERLAY, MUSICAL SYMBOL COMBINING STEM, MUSICAL SYMBOL COMBINING FLAG-1 | Line 27305..27306'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::Same
# Failed test ' (𝆹̴𝅥) MUSICAL SYMBOL SEMIBREVIS WHITE, COMBINING TILDE OVERLAY, MUSICAL SYMBOL COMBINING STEM unicmp  (𝆹̴𝅥𝅮) MUSICAL SYMBOL SEMIBREVIS WHITE, COMBINING TILDE OVERLAY, MUSICAL SYMBOL COMBINING STEM, MUSICAL SYMBOL COMBINING FLAG-1 | Line 27618..27619'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::Same
# Failed test ' (𝆺̴𝅥) MUSICAL SYMBOL SEMIBREVIS BLACK, COMBINING TILDE OVERLAY, MUSICAL SYMBOL COMBINING STEM unicmp  (𝆺̴𝅥𝅮) MUSICAL SYMBOL SEMIBREVIS BLACK, MUSICAL SYMBOL COMBINING STEM, COMBINING TILDE OVERLAY, MUSICAL SYMBOL COMBINING FLAG-1 | Line 27649..27650'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::Same
# Failed test ' (Ă) LATIN CAPITAL LETTER A, COMBINING BREVE unicmp  (ắ) LATIN SMALL LETTER A WITH BREVE, COMBINING ACUTE ACCENT | Line 52366..52367'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Â) LATIN CAPITAL LETTER A, COMBINING CIRCUMFLEX ACCENT unicmp  (ấ) LATIN SMALL LETTER A, COMBINING CIRCUMFLEX ACCENT, COMBINING ACUTE ACCENT | Line 52404..52405'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Å) LATIN CAPITAL LETTER A, COMBINING RING ABOVE unicmp  (å̴) LATIN SMALL LETTER A, COMBINING RING ABOVE, COMBINING TILDE OVERLAY | Line 52449..52450'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ä) LATIN CAPITAL LETTER A, COMBINING DIAERESIS unicmp  (ǟ) LATIN SMALL LETTER A, COMBINING DIAERESIS, COMBINING MACRON | Line 52463..52464'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ȧ) LATIN CAPITAL LETTER A, COMBINING DOT ABOVE unicmp  (ǡ) LATIN SMALL LETTER A, COMBINING DOT ABOVE, COMBINING MACRON | Line 52483..52484'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ạ) LATIN CAPITAL LETTER A, COMBINING DOT BELOW unicmp  (ặ) LATIN SMALL LETTER A, COMBINING BREVE, COMBINING DOT BELOW | Line 52527..52528'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ç) LATIN CAPITAL LETTER C, COMBINING CEDILLA unicmp  (ç̴) LATIN SMALL LETTER C, COMBINING CEDILLA, COMBINING TILDE OVERLAY | Line 56384..56385'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ê) LATIN CAPITAL LETTER E, COMBINING CIRCUMFLEX ACCENT unicmp  (ê̴) LATIN SMALL LETTER E, COMBINING CIRCUMFLEX ACCENT, COMBINING TILDE OVERLAY | Line 57361..57362'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ȩ) LATIN CAPITAL LETTER E, COMBINING CEDILLA unicmp  (ȩ̴) LATIN SMALL LETTER E, COMBINING TILDE OVERLAY, COMBINING CEDILLA | Line 57423..57424'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ē) LATIN CAPITAL LETTER E, COMBINING MACRON unicmp  (ḗ) LATIN SMALL LETTER E WITH MACRON, COMBINING ACUTE ACCENT | Line 57443..57444'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ẹ) LATIN CAPITAL LETTER E, COMBINING DOT BELOW unicmp  (ê̴) LATIN SMALL LETTER E, COMBINING CIRCUMFLEX ACCENT, COMBINING TILDE OVERLAY | Line 57483..57484'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ï) LATIN CAPITAL LETTER I, COMBINING DIAERESIS unicmp  (ḯ) LATIN SMALL LETTER I, COMBINING DIAERESIS, COMBINING ACUTE ACCENT | Line 59425..59426'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ḷ) LATIN CAPITAL LETTER L, COMBINING DOT BELOW unicmp  (ḹ) LATIN SMALL LETTER L, COMBINING MACRON, COMBINING DOT BELOW | Line 60779..60780'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ǿ) LATIN CAPITAL LETTER O WITH STROKE, COMBINING ACUTE ACCENT unicmp  (ó) LATIN SMALL LETTER O, COMBINING ACUTE ACCENT | Line 62356..62357'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ô) LATIN CAPITAL LETTER O, COMBINING CIRCUMFLEX ACCENT unicmp  (ố) LATIN SMALL LETTER O WITH CIRCUMFLEX, COMBINING ACUTE ACCENT | Line 62384..62385'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ö) LATIN CAPITAL LETTER O, COMBINING DIAERESIS unicmp  (ȫ) LATIN SMALL LETTER O, COMBINING DIAERESIS, COMBINING MACRON | Line 62428..62429'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Õ) LATIN CAPITAL LETTER O, COMBINING TILDE unicmp  (ṍ) LATIN SMALL LETTER O, COMBINING TILDE, COMBINING ACUTE ACCENT | Line 62448..62449'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ȯ) LATIN CAPITAL LETTER O, COMBINING DOT ABOVE unicmp  (ȱ) LATIN SMALL LETTER O, COMBINING DOT ABOVE, COMBINING MACRON | Line 62478..62479'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ǫ) LATIN CAPITAL LETTER O, COMBINING OGONEK unicmp  (ō̴) LATIN SMALL LETTER O, COMBINING MACRON, COMBINING TILDE OVERLAY | Line 62492..62493'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ō) LATIN CAPITAL LETTER O, COMBINING MACRON unicmp  (ṓ) LATIN SMALL LETTER O, COMBINING MACRON, COMBINING ACUTE ACCENT | Line 62506..62507'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ơ) LATIN CAPITAL LETTER O, COMBINING HORN unicmp  (ớ) LATIN SMALL LETTER O, COMBINING HORN, COMBINING ACUTE ACCENT | Line 62546..62547'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ọ) LATIN CAPITAL LETTER O, COMBINING DOT BELOW unicmp  (ô̴) LATIN SMALL LETTER O, COMBINING TILDE OVERLAY, COMBINING CIRCUMFLEX ACCENT | Line 62592..62593'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ṛ) LATIN CAPITAL LETTER R, COMBINING DOT BELOW unicmp  (r̴̄) LATIN SMALL LETTER R, COMBINING TILDE OVERLAY, COMBINING MACRON | Line 64076..64077'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ś) LATIN CAPITAL LETTER S, COMBINING ACUTE ACCENT unicmp  (ṥ) LATIN SMALL LETTER S, COMBINING ACUTE ACCENT, COMBINING DOT ABOVE | Line 64636..64637'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Š) LATIN CAPITAL LETTER S, COMBINING CARON unicmp  (ṧ) LATIN SMALL LETTER S, COMBINING CARON, COMBINING DOT ABOVE | Line 64656..64657'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ṣ) LATIN CAPITAL LETTER S, COMBINING DOT BELOW unicmp  (ṣ̴) LATIN SMALL LETTER S, COMBINING DOT BELOW, COMBINING TILDE OVERLAY | Line 64682..64683'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ü) LATIN CAPITAL LETTER U, COMBINING DIAERESIS unicmp  (ǘ) LATIN SMALL LETTER U, COMBINING DIAERESIS, COMBINING ACUTE ACCENT | Line 65708..65709'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ũ) LATIN CAPITAL LETTER U, COMBINING TILDE unicmp  (ṹ) LATIN SMALL LETTER U, COMBINING TILDE, COMBINING ACUTE ACCENT | Line 65752..65753'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ū) LATIN CAPITAL LETTER U, COMBINING MACRON unicmp  (ū̴) LATIN SMALL LETTER U, COMBINING MACRON, COMBINING TILDE OVERLAY | Line 65772..65773'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ư) LATIN CAPITAL LETTER U, COMBINING HORN unicmp  (ứ) LATIN SMALL LETTER U, COMBINING ACUTE ACCENT, COMBINING HORN | Line 65804..65805'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἀ) GREEK CAPITAL LETTER ALPHA, COMBINING COMMA ABOVE unicmp  (ἀ̴) GREEK SMALL LETTER ALPHA, COMBINING GREEK KORONIS, COMBINING TILDE OVERLAY | Line 68793..68794'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἁ) GREEK CAPITAL LETTER ALPHA, COMBINING REVERSED COMMA ABOVE unicmp  (ἁ̴) GREEK SMALL LETTER ALPHA, COMBINING REVERSED COMMA ABOVE, COMBINING TILDE OVERLAY | Line 68855..68856'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἐ) GREEK CAPITAL LETTER EPSILON, COMBINING COMMA ABOVE unicmp  (ἐ̴) GREEK SMALL LETTER EPSILON, COMBINING TILDE OVERLAY, COMBINING GREEK KORONIS | Line 69493..69494'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἑ) GREEK CAPITAL LETTER EPSILON, COMBINING REVERSED COMMA ABOVE unicmp  (ἑ̴) GREEK SMALL LETTER EPSILON, COMBINING REVERSED COMMA ABOVE, COMBINING TILDE OVERLAY | Line 69515..69516'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἠ) GREEK CAPITAL LETTER ETA, COMBINING COMMA ABOVE unicmp  (ἤ) GREEK SMALL LETTER ETA, COMBINING GREEK KORONIS, COMBINING ACUTE ACCENT | Line 69845..69846'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἡ) GREEK CAPITAL LETTER ETA, COMBINING REVERSED COMMA ABOVE unicmp  (ἡ̴) GREEK SMALL LETTER ETA, COMBINING REVERSED COMMA ABOVE, COMBINING TILDE OVERLAY | Line 69907..69908'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἰ) GREEK CAPITAL LETTER IOTA, COMBINING COMMA ABOVE unicmp  (ἰ̴) GREEK SMALL LETTER IOTA, COMBINING TILDE OVERLAY, COMBINING GREEK KORONIS | Line 70413..70414'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ἱ) GREEK CAPITAL LETTER IOTA, COMBINING REVERSED COMMA ABOVE unicmp  (ἱ̴) GREEK SMALL LETTER IOTA WITH DASIA, COMBINING TILDE OVERLAY | Line 70444..70445'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ϊ) GREEK CAPITAL LETTER IOTA, COMBINING DIAERESIS unicmp  (ΐ) GREEK SMALL LETTER IOTA WITH DIALYTIKA AND TONOS | Line 70502..70503'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ὀ) GREEK CAPITAL LETTER OMICRON, COMBINING COMMA ABOVE unicmp  (ὀ̴) GREEK SMALL LETTER OMICRON, COMBINING TILDE OVERLAY, COMBINING COMMA ABOVE | Line 71164..71165'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ὁ) GREEK CAPITAL LETTER OMICRON, COMBINING REVERSED COMMA ABOVE unicmp  (ὁ̴) GREEK SMALL LETTER OMICRON, COMBINING TILDE OVERLAY, COMBINING REVERSED COMMA ABOVE | Line 71186..71187'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ὐ) GREEK SMALL LETTER UPSILON, COMBINING COMMA ABOVE unicmp  (ὔ) GREEK SMALL LETTER UPSILON, COMBINING COMMA ABOVE, COMBINING ACUTE ACCENT | Line 71827..71828'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ὑ) GREEK CAPITAL LETTER UPSILON, COMBINING REVERSED COMMA ABOVE unicmp  (ὕ) GREEK SMALL LETTER UPSILON, COMBINING REVERSED COMMA ABOVE, COMBINING ACUTE ACCENT | Line 71845..71846'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ϔ) GREEK UPSILON WITH HOOK SYMBOL, COMBINING DIAERESIS unicmp  (ΰ) GREEK SMALL LETTER UPSILON WITH DIALYTIKA AND TONOS | Line 71907..71908'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ὠ) GREEK CAPITAL LETTER OMEGA, COMBINING COMMA ABOVE unicmp  (ὠ̴) GREEK SMALL LETTER OMEGA, COMBINING GREEK KORONIS, COMBINING TILDE OVERLAY | Line 72388..72389'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (Ὡ) GREEK CAPITAL LETTER OMEGA, COMBINING REVERSED COMMA ABOVE unicmp  (ὥ) GREEK SMALL LETTER OMEGA WITH DASIA, COMBINING ACUTE ACCENT | Line 72451..72452'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎שׁ‎‎) HEBREW LETTER SHIN WITH SHIN DOT unicmp  (‎‎שּ‎‎) HEBREW LETTER SHIN WITH DAGESH | Line 78579..78580'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎שׁ‎‎) HEBREW LETTER SHIN WITH SHIN DOT unicmp  (‎‎שּ‎‎) HEBREW LETTER SHIN WITH DAGESH | Line 78585..78586'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎שׁ‎‎) HEBREW LETTER SHIN WITH SHIN DOT unicmp  (‎‎שּ‎‎) HEBREW LETTER SHIN WITH DAGESH | Line 78594..78595'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎שׁ‎‎) HEBREW LETTER SHIN WITH SHIN DOT unicmp  (‎‎שּ‎‎) HEBREW LETTER SHIN WITH DAGESH | Line 78603..78604'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎آ֑‎‎) ARABIC LETTER ALEF, HEBREW ACCENT ETNAHTA, ARABIC MADDAH ABOVE unicmp  (‎‎آ‎‎) ARABIC LETTER ALEF WITH MADDA ABOVE | Line 78902..78903'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎أ֑‎‎) ARABIC LETTER ALEF, HEBREW ACCENT ETNAHTA, ARABIC HAMZA ABOVE unicmp  (‎‎أ‎‎) ARABIC LETTER ALEF WITH HAMZA ABOVE | Line 78927..78928'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎ؤ֑‎‎) ARABIC LETTER WAW, HEBREW ACCENT ETNAHTA, ARABIC HAMZA ABOVE unicmp  (‎‎ؤ‎‎) ARABIC LETTER WAW WITH HAMZA ABOVE | Line 78972..78973'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (‎‎ئ֑‎‎) ARABIC LETTER YEH, HEBREW ACCENT ETNAHTA, ARABIC HAMZA ABOVE unicmp  (‎‎ئ‎‎) ARABIC LETTER YEH WITH HAMZA ABOVE | Line 79040..79041'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ཱི𝅥) TIBETAN VOWEL SIGN AA, MUSICAL SYMBOL COMBINING STEM, TIBETAN VOWEL SIGN I unicmp  (ཱི) TIBETAN VOWEL SIGN AA, TIBETAN VOWEL SIGN I | Line 106131..106132'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ཱྀ𝅥) TIBETAN VOWEL SIGN AA, MUSICAL SYMBOL COMBINING STEM, TIBETAN VOWEL SIGN REVERSED I unicmp  (ཱྀ) TIBETAN VOWEL SIGN AA, TIBETAN VOWEL SIGN REVERSED I | Line 106154..106155'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ཱུ𝅥) TIBETAN VOWEL SIGN AA, MUSICAL SYMBOL COMBINING STEM, TIBETAN VOWEL SIGN U unicmp  (ཱུ) TIBETAN VOWEL SIGN AA, TIBETAN VOWEL SIGN U | Line 106177..106178'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ཱུ) TIBETAN VOWEL SIGN AA, TIBETAN VOWEL SIGN U unicmp  (ྲྀ) TIBETAN VOWEL SIGN VOCALIC R | Line 106181..106182'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ྲྀ𝅥) TIBETAN SUBJOINED LETTER RA, MUSICAL SYMBOL COMBINING STEM, TIBETAN VOWEL SIGN REVERSED I unicmp  (ྲྀ) TIBETAN VOWEL SIGN VOCALIC R | Line 106192..106193'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ྲྀ) TIBETAN VOWEL SIGN VOCALIC R unicmp  (ྲཱྀ) TIBETAN VOWEL SIGN VOCALIC R, TIBETAN VOWEL SIGN AA | Line 106196..106197'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ྲཱྀ) TIBETAN SUBJOINED LETTER RA, TIBETAN VOWEL SIGN AA, TIBETAN VOWEL SIGN REVERSED I unicmp  (ླྀ) TIBETAN VOWEL SIGN VOCALIC L | Line 106221..106222'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ླྀ𝅥) TIBETAN SUBJOINED LETTER LA, MUSICAL SYMBOL COMBINING STEM, TIBETAN VOWEL SIGN REVERSED I unicmp  (ླྀ) TIBETAN VOWEL SIGN VOCALIC L | Line 106232..106233'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (ླྀ) TIBETAN VOWEL SIGN VOCALIC L unicmp  (ླཱྀ) TIBETAN VOWEL SIGN VOCALIC L, TIBETAN VOWEL SIGN AA | Line 106236..106237'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (가) HANGUL CHOSEONG KIYEOK, HANGUL JUNGSEONG A unicmp  (ᄀ𝅥ᅡ) HANGUL CHOSEONG KIYEOK, MUSICAL SYMBOL COMBINING STEM, HANGUL JUNGSEONG A | Line 128056..128057'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (기) HANGUL CHOSEONG KIYEOK, HANGUL JUNGSEONG I unicmp  (ᄀ𝅥ᅵ) HANGUL CHOSEONG KIYEOK, MUSICAL SYMBOL COMBINING STEM, HANGUL JUNGSEONG I | Line 128110..128111'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (하) HANGUL CHOSEONG HIEUH, HANGUL JUNGSEONG A unicmp  (ᄒ𝅥ᅡ) HANGUL CHOSEONG HIEUH, MUSICAL SYMBOL COMBINING STEM, HANGUL JUNGSEONG A | Line 128530..128531'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (히) HANGUL CHOSEONG HIEUH, HANGUL JUNGSEONG I unicmp  (ᄒ𝅥ᅵ) HANGUL CHOSEONG HIEUH, MUSICAL SYMBOL COMBINING STEM, HANGUL JUNGSEONG I | Line 128541..128542'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (𘴈) TANGUT IDEOGRAPH-18D08 unicmp  (𛅰) NUSHU CHARACTER-1B170 | Line 166028..166029'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
# Failed test ' (𘳕) KHITAN SMALL SCRIPT CHARACTER-18CD5 unicmp  (一) CJK UNIFIED IDEOGRAPH-4E00 | Line 166063..166064'
# at CollationTestGen.p6 line 31
# expected: Order::Less
#      got: Order::More
Failed: 27306, 27619, 27650, 52367, 52405, 52450, 52464, 52484, 52528, 56385, 57362, 57424, 57444, 57484, 59426, 60780, 62357, 62385, 62429, 62449, 62479, 62493, 62507, 62547, 62593, 64077, 64637, 64657, 64683, 65709, 65753, 65773, 65805, 68794, 68856, 69494, 69516, 69846, 69908, 70414, 70445, 70503, 71165, 71187, 71828, 71846, 71908, 72389, 72452, 78580, 78586, 78595, 78604, 78903, 78928, 78973, 79041, 106132, 106155, 106178, 106182, 106193, 106197, 106222, 106233, 106237, 128057, 128111, 128531, 128542, 166029, 166064
# You failed 72 tests of 202763

Reply:
Yeah those look like old failures to me.